ykrt/src/compile/jitc_yk/opt/lvn.rs

```diff
@@ -0,0 +1,115 @@
+#[cfg(test)]
+use crate::compile::jitc_yk::jit_ir::ConsumeInst;
+use crate::compile::{
+    jitc_yk::jit_ir::{AddInst, Inst, InstIdx, Module, Operand},
+    CompilationError,
+};
+use typed_index_collections::TiVec;
+
+/// Perform (very very) simple dead-code elimination.
+pub(super) fn lvn(mut m: Module) -> Result<Module, CompilationError> {
+    // Which operations are used?
```

Comment:
s/operations/operands?/

Reply:
Should actually be "instructions"!

Reply:
"Instructions" makes the most sense IMHO.

We are scanning for local variable operands, and then caching a list of instructions which define those operands.

Reply:
Fixed in eeccf229.